src/BloomExe/web/controllers/BookSettingsApi.cs

```diff
@@ -54,12 +60,36 @@ private  void HandleBookSettings(ApiRequest request)
 
 		private bool GetIsBookATemplate()
 		{
-			return false;
+			if (_bookSelection == null || _bookSelection.CurrentSelection == null || _bookSelection.CurrentSelection.BookInfo == null)
+				return false;
+			var info = _bookSelection.CurrentSelection.BookInfo;
+			SaveOriginalTypeIfNecessary(info);
+			return info.Type == Book.Book.BookType.Template;
 		}
 
 		private void UpdateBookTemplateMode(bool isTemplateBook)
 		{
-			
+			if (_bookSelection == null || _bookSelection.CurrentSelection == null || _bookSelection.CurrentSelection.BookInfo == null)
+				return;     // sheer paranoia
+			var info = _bookSelection.CurrentSelection.BookInfo;
+			SaveOriginalTypeIfNecessary(info);
+			if (isTemplateBook)
+			{
+				info.Type = Book.Book.BookType.Template;
+			}
+			else
+			{
+				Book.Book.BookType type = _bookOriginalType[info];
+				if (type == Book.Book.BookType.Template)
+					type = Book.Book.BookType.Publication;
```

Comment:
I need to look into what it would mean to always revert to "Publication"; the book could have been "Shell", turned into template, Bloom restart, then unchecked, now it's a "Publication".